.github/workflows/wasm.yml

```
@@ -0,0 +1,33 @@
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+  CARGO_TARGET_WASM32_UNKNOWN_UNKNOWN_RUNNER: wasm-bindgen-test-runner
+
+jobs:
+  test-wasm:
+
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)